bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -1699,7 +1717,8 @@ void init(int width, int height) {
 	}
 	type = SWT.BITMAP;
 	long hDC = device.internal_new_GC(null);
-	handle = OS.CreateCompatibleBitmap(hDC, width, height);
+	setHandleForZoomLevel(OS.CreateCompatibleBitmap(hDC, width, height), height, width, getCurrentDeviceZoom());
+	handleMap.put(getCurrentDeviceZoom(), handle);
```

Comment:
Isn't setHandleForZoomLevel adding it already to the handleMap?

Reply:
yeah, a miss. Sorry